docs/Recommendations.md

```diff
@@ -8,21 +8,20 @@ Fastify.
 * [Use A Reverse Proxy](#reverseproxy)
 * [Kubernetes](#kubernetes)
 
-## Use A Reverse Proxy
+## Use a Reverse Proxy
 <a id="reverseproxy"></a>
 
 Node.js is an early adopter of frameworks shipping with an easy to use web
 server within the standard library. Previously, with languages like PHP or
 Python, one would need either a web server with specific support for the
 language or the ability to setup some sort of [CGI gateway][cgi] that works
-with the language. With Node.js, one can simply write an application that
+with the language. With Node.js, one can write an application that
 _directly_ handles HTTP requests. As a result, the temptation is to write
 applications that handle requests for multiple domains, listen on multiple
-ports (i.e. HTTP _and_ HTTPS), and various other scenarios and combinations
-thereof. Further, the temptation is to then expose these applications directly
+ports (i.e. HTTP _and_ HTTPS), and then expose these applications directly
 to the Internet to handle requests.
 
-The Fastify team **strongly** considers this to be an anti-pattern and extremely
+The Fastify team **strongly** considers this an anti-pattern and extremely
```

Comment:
Why?

Reply:
Sentence is still grammatically correct without "to be", and reduces the wordiness. 😬

Reply:
But does it have the same intention without "to be"?

Reply:
After running this by my partner, who is an English teacher, she point blank said "to be" reads better.

...I expect to be single shortly.

Will revert!